test/unit/pct_encoding.cpp

```diff
@@ -362,7 +362,7 @@ class pct_encoding_test
         auto const check =
             []( string_view s,
                 string_view m0,
-                bool space_to_plus = false)
```

Comment:
why did we remove the default argument?

Reply:
```
warning: default argument specified for lambda parameter
```

Reply:
can we just make this a separate function, to keep the default?

Reply:
Sure. It's not that nice to have long lambdas anyway.